docker-compose.yml

```diff
@@ -1,28 +1,28 @@
-version: "3.4"
-services:
-  graphelier-app:
-    build:
-      context: ./app
-      target: dev
-    ports:
-      - "3000:3000"
-    depends_on:
-      - "graphelier-service"
-  graphelier-service:
-    build:
-      context: ./core
-      target: prod
-    ports:
-      - "5050:5050"
-    volumes:
-      - ./core/graphelier-service:/go/src/graphelier/core/graphelier-service
-    depends_on:
-      - "mongo"
-  mongo:
-    build:
-      context: ./core/db
-    container_name: "graphelier-db"
-    ports:
-      - "27017:27017"
-    volumes:
-      - "./core/db/data:/data/db"
+version: "3.4"

```

Comment:
Line-endings issue in this file?

Reply:
I guess there was an issue for this one as well. Is there a way for me to remove it from being "modified"

Reply:
You can try doing a `git checkout origin/master -- docker-compose.yml` or something similar